webapp/src/components/CollectionCardWide.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import CreateIcon from "./CreateIcon";
+import {
+  ICON_TYPES_LIST,
+  INNER_ICON_TYPES_LIST,
+} from "../constants/iconTypeConsts";
+
+export default function CollectionCardWide({
+  title,
+  iconVariant,
+  innerIconVarient,
+  bgColour,
+  innerIconBg,
+  customStyles,
```

Comment:
Check if we can add a variant to the `CollectionCard` and achieve this behavior through changing classes. Else apply the same changes  requested as in `CollectionCard.jsx`